index.html

```diff
@@ -0,0 +1,189 @@
+                </p>
+                <br>
+                <br>
+                <p>
+                    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quia veniam, recusandae reiciendis dolorem amet eaque eum itaque autem ad voluptatum, accusamus quos molestiae officiis nemo laborum et provident numquam beatae. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Doloremque quasi delectus minima dolorem enim ullam debitis cupiditate hic harum laudantium sapiente, error, quos maxime tempora quod nemo odio voluptate saepe.
+                </p>
+            </div>
+        </div>
+    </section> -->
+    <section class="section5">
+        <div class="left-half grid">
+            <h1>Sponsors</h1>
+            <ul class="sponsor-list">
+                <li id="benq">
+                    <div class="img-wrapper"><img src="public/img/benq.png" alt=""></div>
+                    <p>BenQ</p>
+                </li>
+                <li id="sostronk">
+                    <div class="img-wrapper"><img src="public/img/sostronk.png" alt=""></div>
+                    <p>Sostronk</p>
+                </li>
+                <li id="the_source">
+                    <div class="img-wrapper"><img src="public/img/the_source.jpeg" alt=""></div>
+                    <p>The Source</p>
+                </li>
+                <li id="gaming_central">
+                    <div class="img-wrapper"><img src="public/img/gc.png" alt=""></div>
+                    <p>Gaming Central</p>
+                </li>
+            </ul>
+        </div>
+        <div class="right-half grid">
+            <div class="container-fluid">
+                <h1>Contacts</h1>
+            </div>
+            <ul class="contact-list">
+                <li>
+                    <h1>Rahul Bisht</h1>
+                    <h2>Event Coordinator</h2>
+                    <h3>9999911628</h3>
+                    <h4>rahulbishtbits@gmail.com</h4>
+                </li>
+                <li>
+                    <h1>Yash Singhal</h1>
+                    <h2>Event Coordinator</h2>
+                    <h3>7722047558</h3>
+                    <h4>singhal.yash35@gmail.com</h4>
+                </li>
+                <li>
+                    <h1>Himanshu Lakhotiya</h1>
+                    <h2>Event Coordinator</h2>
+                    <h3>9970735246</h3>
+                    <h4>himanshu.lakhotiya@gmail.com</h4>
+                </li>
+                <li>
+                    <h1>Chetan Tiwari</h1>
+                    <h2>Event Coordinator</h2>
+                    <h3>9130651218</h3>
+                    <h4>ctiwari409@gmail.com</h4>

```

Comment:
I checked them locally, I feel there is problem with the size. The feature over here looks cool. But the names are overflowing the parent div